governance/chair-proposal-process.md

```diff
@@ -0,0 +1,148 @@
+# Environmental Sustainability Leadership Team
+
+The leadership team of the WG are servant leads. They are an enabler for the group and support the creation and success of the different activities. They are the driver for the group for not losing the higher targets out of sight and taking decisions when no consensus can be found.
+
+## Team Structure
+
+The leadership team is conducted of:
+* Three Co-Chairs
+* Three Tech Leads
+
+As WG we strive to be a gender, company and geolocation, diverse group. 
```

Comment:
It is a good start to have diversity explicitly spelled out in the first place.

Reply:
"as a TAG we strive..."